template/v1/dirs/usr/local/bin/start-jupyter-server

```diff
@@ -6,8 +6,22 @@ eval "$(micromamba shell hook --shell=bash)"
 # Activate conda environment 'base', which is the default environment for Cosmos
 micromamba activate base
 
+# Start Jupyter server in rtc mode for shared spaces
+if [ -n "$SAGEMAKER_APP_TYPE_LOWERCASE" ] && [ -n "$SAGEMAKER_SPACE_TYPE_LOWERCASE" == "shared" ]; then
+  # SAGEMAKER_APP_TYPE is set, indicating the server is running within a SageMaker
+  # app. Configure the base url to be `/<app-type-in-lower-case>/default`. 
+  # SAGEMAKER_SPACE_TYPE_LOWERCASE flag is used to determine if the server should start 
+  # in real-time-collaboration mode for a given space.
+  jupyter lab --ip 0.0.0.0 --port 8888 \
+    --ServerApp.base_url="/$SAGEMAKER_APP_TYPE_LOWERCASE/default" \
+    --ServerApp.token='' \
+    --ServerApp.allow_origin='*' \
+    --collaborative \
+    --ServerApp.identity_provider_class=sagemaker_jupyterlab_extension_common.identity.SagemakerIdentityProvider \
+    --YDocExtension.ystore_class=sagemaker_jupyterlab_extension_common.ydoc_override.ydoc.MySQLiteYStore \
```

Comment:
nit- can you remove the trailing backslash at end of this line?

Reply:
+1 on this - the trailing backslash is not needed here.